telegram/_chat.py

```diff
@@ -355,6 +359,92 @@ def de_json(cls, data: Optional[JSONDict], bot: "Bot") -> Optional["Chat"]:
 
         return super()._de_json(data=data, bot=bot, api_kwargs=api_kwargs)
 
+    def mention_markdown(self, name: str = None) -> str:
```

Comment:
Just wondering if we still need to support V1 markdown in new shortcuts...

Reply:
forgot to reply to this.. yes, I think we should. In many cases it's less annoying than MDv2

Reply:
I think we meet the decision that drop markdown v1 supports.

Reply:
I think there is a misunderstanding - I meant to say we should be supporting MarkdownV1 and thus mention_markdown shouldn't be removed imo

Reply:
I'm with harshil here